docs/usage.md

```diff
@@ -24,32 +24,30 @@ nextflow run main.nf \
 
 ### Full samplesheet
 
-A final samplesheet file consisting of both single- and paired-end data may look something like [this example samplesheet](../tests/data/grzqc_samplesheet.csv). This is for 1 sample, which has been sequenced four times.
```

Comment:
This file doesn't exist, so I deleted this line. Let me know if I should point it to a different link?

Reply:
Ok I'll update it!